src/libs/fileDownload/FileUtils.ts

```diff
@@ -135,34 +122,29 @@ function appendTimeToFileName(fileName) {
 
 /**
  * Reads a locally uploaded file
- *
- * @param {String} path - the blob url of the locally uplodaded file
- * @param {String} fileName
- * @returns {Promise}
  */
-const readFileAsync = (path, fileName) =>
+const readFileAsync: ReadFileAsync = (path, fileName) =>
```

Comment:
```suggestion
/**
 * Reads a locally uploaded file
 *
 * @param path - the blob url of the locally uploaded file
 */
const readFileAsync: ReadFileAsync = (path, fileName) =>
```

we can keep comment for this variable